.azure-pipelines/azure-pipelines-build-vs-and-test.yml

```diff
@@ -0,0 +1,135 @@
+pr:
```

Comment:
Is this pipeline supposed to be executed for each PR or be manually executed?

Reply:
It's manually executed, maybe need to set to pr: none